src/App.tsx

```diff
@@ -1,61 +1,48 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { Todo } from './types/Todo';
+import { getUsersById } from './utils/getUsersById';
+
+const initialTodos: Todo[] = todosFromServer.map(todo => {
+  const preparingUser = getUsersById(todo.userId);
```

Comment:
The `getUsersById` function might return `undefined` if no user is found with the given `userId`. This could cause issues when rendering components that expect a valid user object. Consider adding a check or providing a default value to handle cases where the user is not found.